app/Member.php

```diff
@@ -22,10 +22,20 @@ class Member extends Model
      *
      * @var array
      */
-    protected $fillable = array(
```

Comment:
I thought that all other attributes not listed in guarded would be fillable automatically so I removed it. I just tested the signup and yes, its broken for some strange reason. Will try to figure it out.

```preg_replace(): Parameter mismatch, pattern is a string while replacement is an array```